### PR TITLE
Config doc update

### PR DESCRIPTION
**Pull request by mdurrant-b3**
First pass, opening this so folks can comment.  The header levels need some attention but that can be addressed after the actual content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +209,47 @@ Authentication credentials to control the A10 appliance via the AXAPI.
...
+#### vThunder License Manager Configuration
+The A10 vThunder virtual load balancing appliance has a flexible system for licensing.  Below is a sample configuration for license management (stored in `/etc/a10/config.py`):
+```python
+license_manager = {
+        "hosts": [
+                {"ip": "pdx.a10cloud.com", "port": 443},
+                {"ip": "sfo.a10cloud.com", "port": 443},
+                {"ip": "iad.a10cloud.com", "port": 443}
+        ],
+        "serial": "SNE64DD9823105C4F7E02B6AF4026C94",
```
Comment (dougwig) [inline README.md]: if that's a real key, mangle it.

**Comment by mdurrant-b3** [comment]
Random CI failure - I don't want to re-run CI for a README.

File: README.md [hunk 2]
```diff
@@ -134,6 +209,47 @@ Authentication credentials to control the A10 appliance via the AXAPI.
 Version of the A10 appliance's AXAPI. `"2.1"` for 2.X series ACOS versions,
 `"3.0"` for 4.X versions.
 
+
+#### vThunder License Manager Configuration
+The A10 vThunder virtual load balancing appliance has a flexible system for licensing.  Below is a sample configuration for license management (stored in `/etc/a10/config.py`):
+```python
+license_manager = {
```
Comment (Cedev) [inline README.md]: license_manager is per vthunder

**Comment by Cedev** [comment]
Became/superseded by PR #338

**Pull request closed without merging**